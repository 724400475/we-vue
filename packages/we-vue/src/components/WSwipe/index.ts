import Swipe from './swipe'

export { Swipe }
export default Swipe
