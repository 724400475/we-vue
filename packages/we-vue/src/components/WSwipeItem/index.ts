import SwipeItem from './swipe-item'

export { SwipeItem }
export default SwipeItem
