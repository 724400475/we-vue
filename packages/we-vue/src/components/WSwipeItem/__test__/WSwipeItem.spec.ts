import { mount } from '@vue/test-utils'
import WSwipe from '../../WSwipe'
import WSwipeItem from '../WSwipeItem'

describe('swipe item', () => {
  test('create', () => {
    const wrapper = mount(WSwipe, {
      attachToDocument: true,
      slots: {
        default: [WSwipeItem, WSwipeItem],
      },
    })

    expect(wrapper.findAll(WSwipeItem)).toHaveLength(2)
    expect(wrapper.find(WSwipeItem).classes()).toContain('wv-swipe-item')

    expect(wrapper.html()).toMatchSnapshot()
  })
})
