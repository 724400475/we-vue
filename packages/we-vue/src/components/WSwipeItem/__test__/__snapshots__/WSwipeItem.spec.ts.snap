// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`swipe item create 1`] = `
<div class="wv-swipe">
  <div class="wv-swipe__wrapper" style="padding-left: 0px; width: 100%; transition-duration: 0ms; transform: translate3d(0px, 0, 0);">
    <div class="wv-swipe-item" style="width: 0px; transform: translate3d(0px, 0, 0);"></div>
    <div class="wv-swipe-item" style="width: 0px; transform: translate3d(0px, 0, 0);"></div>
  </div>
  <!---->
</div>
`;
